test/cql-pytest/test_wasm.py

```diff
@@ -948,3 +965,6 @@ def test_word_double(cql, test_keyspace, table1, scylla_with_wasm_only):
         cql.execute(f"INSERT INTO {table} (p, txt) VALUES (1001, 'cat42')")
         res = [row for row in cql.execute(f"SELECT {test_keyspace}.{dbl_name}(txt) AS result FROM {table} WHERE p = 1001")]
         assert len(res) == 1 and res[0].result == 'cat42cat42'
+
```

Comment:
Need tests to verify reuse. You can add a metric for function creations.

Reply:
Metrics are a great idea, for the sake of future benchmarking too - e.g. we could verify how many times functions were compiled vs run, how many wasm-induced allocations were performed, and so on.

Reply:
It's the same idea behind prepared statements and their metrics.

Reply:
One frustrating problem with tests that use global metrics is that they cannot be run reliably in parallel against the same instance of Scylla. It's not a real problem currently (we *don't* run tests in parallel), so maybe there's no reason to be apprehensive about it. However, note that there is a reliable way to have metrics in parallel tests - it is to use per-table metrics. It turns out (!) we also have a way to extract per-table metrics via the rest api - see rest_api.py::get_column_family_metric. We don't use this trick enough in tests. We should.